bioio/bio_image.py

```diff
@@ -1003,6 +1003,14 @@ def time_interval(self) -> biob.types.TimeInterval:
         """
         return self.reader.time_interval
 
+    @property
+    def standard_metadata(self) -> biob.standard_metadata.StandardMetadata:
+        """
+        Return the manifest of standard metadata as a dictionary. The possible
```

Comment:
Not a dictionary so maybe this?
```suggestion
        Return a set of standardized metadata. The possible
```

Reply:
whoops! relic of the past